.travis.yml

```diff
@@ -42,7 +42,8 @@ notifications:
     - joshua.suereth@typesafe.com
 
 script:
-  - sbt -J-XX:ReservedCodeCacheSize=128m "$SCRIPTED_TEST"
+  - sbt ";publishLocal;project bundledLauncherProj;publishLocal;project precompiled-2_9_2;publishLocal;project precompiled-2_8_2;publishLocal;project precompiled-2_9_3;publishLocal"     
```

Comment:
It will be required only - when 0.13.9 will be published. After it the .travis script can be restored. 
Please checkout the code beaf39a196b530e335b44806069b3686dca43f38 and run the tests. You will receive: errors, see all the builds in: https://travis-ci.org/sbt/sbt/builds/54919087:
```
....
About to run tests:

* actions/*

java.lang.NoSuchMethodException: sbt.test.ScriptedTests$.run(java.io.File, boolean, [Ljava.lang.String;, java.io.File, [Ljava.lang.String;, scala.Function1)

at java.lang.Class.getMethod(Class.java:1667)

at Scripted$.doScripted(Scripted.scala:66)

at $0e28670edea040be68f9$$anonfun$scriptedTask$2$$anonfun$apply$2$$anonfun$apply$3.apply(build.sbt:474)

at $0e28670edea040be68f9$$anonfun$scriptedTask$2$$anonfun$apply$2$$anonfun$apply$3.apply(build.sbt:471)

at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
...
```
I guess it is problem with classloader. Mentioned in stack method exist.
As the solution the new process should be run - by scala 2.10.5 - and then all is run by the same classloader.